test2.rb

```diff
@@ -0,0 +1,26 @@
+a = "bbb"
+
+def foo
+    'a'
```

Comment:
__[development][rubocop]__
* Use 2 (not 4) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

<!-- Comment by [SideCI](https://sideci.com) -->